### DISCUSSION TITLE
Trigger UAC only when needed

### DISCUSSION THREAD

**Discussion by Otiel** [snappysnappydog/Simple-IP-Config#190] [Ideas]
Currently, UAC is triggered when starting the application:

![image](https://user-images.githubusercontent.com/3851902/31494955-d2bbbce4-af55-11e7-8128-e8158c3006de.png)

I've set up the application to start with Windows (with a shortcut in the Startup folder) and to start in the system tray (thanks to your awesome settings :) ), so that the application is ready and available when I need it. Thus, I have the UAC is triggered at Windows startup, which is a bit annoying.

Could you change the code so that the UAC is triggered only when needed, which should be - if I guess correctly - when a profile is applied?

**Comment by KurtisLiggett** [comment]
This turned out to be trickier than expected, but I think I came up with a decent solution.
Hopefully, it will be ready in version 2.9.1.

**Comment by KurtisLiggett** [comment]
I thought I had implemented this, but apparently lost the code changes. Converting this issue to a discussion idea for now.

**Comment by elliepickard** [comment]
Hi, 
I'm having the same issue, is there a resolution? I'm trying to package it so multiple users can use the software without the UAC prompt popping up.

**Reply by KurtisLiggett** [reply]
I will revisit this and try to get this implemented.
